src/taipy/core/config/config.schema.json

```diff
@@ -12,6 +12,10 @@
       "required": [],
       "additionalProperties": {
         "properties": {
+          "core_version": {
```

Comment:
I thought it would simply be `version` ?

Reply:
I wanted to at first, but in core_section, we also have version_number that's for the version management system so the name is confusing